frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx

```diff
@@ -120,7 +120,7 @@ export const MissionQueueCard = ({ order, mission, onDeleteMission }: MissionQue
     const [confirmDeleteDialogOpen, setConfirmDeleteDialogOpen] = useState<boolean>(false)
     const fillColor = tokens.colors.infographic.primary__energy_red_21.hex
     let numberOfTasks = 0
-    mission.tasks.forEach((task) => (numberOfTasks += task.inspections.length))
+    mission.tasks.forEach((task) => (numberOfTasks += 1))
```

Comment:
just use tasks.length?

Reply:
Nope- that is a loop my solution gives correct answer. You can test both ways.

Reply:
Removed loop and assigned const directly now =)